assets/css/table.css

```diff
@@ -60,10 +65,23 @@
   @apply bg-red-200 bg-opacity-20;
 }
 
+<<<<<<< HEAD
 .instructor_dashboard_table td:has(div.highlight-exception) {
   @apply bg-blue-100 rounded-md bg-opacity-50;
 }
 
 div:has(.instructor_dashboard_table) #header_paging {
   @apply sticky left-0;
+=======
+div:has(.remix_materials_table) {
+  @apply overflow-x-hidden;
+}
+
+.remix_materials_table {
+  @apply instructor_dashboard_table border-none;
+}
+
+.remix_materials_table th:last-of-type {
+  text-align: right;
+>>>>>>> e6ee73b5a5 (Add ability to add orphan pages to section curriculum)
```

Comment:
If I'm not mistaken, on lines 75 and 86 is the text that VSCode shows when conflicts need to be resolved, I'm right?

Reply:
Yeah, I don't know how I missed that 🤔